GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared inventory. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointman's outfit, press `O` (if you want to see the pointman's
+personal inventory outfit and manipulate items stored there) or `i` (if
+you want to stash something in the shared inventory from the pointman's
+outfit).
+To open the shared inventory stash, press `I`.
+To open the whole team's belongings (personal and stashed), press `T`.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press `G`.
+
+To pick up an item from the ground to the pointman's outfit, press `g`
+or `,`. Items that have some direct use (melee weapon or armor) 
+will be equipped in the personal equipment outfit while if it has no 
+direct use or the equippment outfit is full, the item will be moved
+to the shared inventory.
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and `RET` buttons to get the item
+management menu. You may want to choose all items from the particular 
+location by pressing `*`.
+
+You may want to cycle among the characters' inventory outfits without leaving 
+the inventory menu. To do that, you may press `<` or `>` to check different 
+inventories: character's outfit, stashed inventory, objects on the ground, etc. 
+
+Quite often, you will need to modify your environment somehow. There are
+numerous ways how to do that, so try and experiment!
+You may craft items, change the terrain, cook food, upgrade or change items,
+and do many other things!
+
+To craft or modify items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the workshop station, drop
+the components, stand on them, press `M`, and point to the direction
+of the item-modifying tile. The components will be automatically consumed,
+and the resulting item will appear. 
+For example, you may want to craft a club from a wooden stick and several 
+steel scraps. Approach a workshop, drop the components, press `M`, and 
+point to the workshop.
+Some crafting recipes may be found by pressing the `~` command while looking
+at the terrain you want to use.
```

Comment:
I'd drop "Some", because all can be found in this way.

Perhaps it's worth it to preserve a couple of details from this removed text? E.g, here mention that "the lore menu, terrain submenu" shows the recipes for encountered terrain.

Stand adjacent to the workshop terrain (blue colon on the map) and experiment. Note that walkable positions can't be activated by bumping and, anyway, crafting requires the Modify command (key M) even for terrain that could be bumped into. Other crafting spots (e.g., any water and fire) work similarly, but the recipes differ. Read the recipes in the lore menu, terrain submenu, or by pressing ~ when crosshair points at terrain that enables crafting.

Reply:
I meant the (repaired) spacesuit. I've just checked, it's there among the recipes of the bench of crafting (the workshop). I do think no are secret in the sense that the recipe is not visible to the user. However, some are hard to craft and until you craft the first specimen, the item does not show in the item lore, so its properties are secret.

Reply:
Regarding line the whole crafting subsection, I just wondered aloud whether some of the old text would be useful, since not all is already contained in the new text.